src/utils/number.cljs

```diff
@@ -64,10 +64,13 @@
   Example usage:
   (convert-to-whole-number 12345 2) ; => 123.45"
   [amount decimals]
-  (-> amount
-      (/ (Math/pow 10 decimals))
-      (.toFixed decimals)
-      remove-trailing-zeroes))
+  (let [amount-bn  (money/->bignumber amount)
```

Comment:
Here you can benefit from `->bignumbers` function defined in `utils.money` it give a convenient way to use `when-let` along with conversion multiple parameters to `bignumber`

Reply:
Thanks @vkjr, looks much cleaner now